### PR TITLE
Enabled xml doc generation

### PR DESCRIPTION
**Pull request by ajaybhargavb**
@NTaylorMullen

### CODE REVIEW

File: src/Microsoft.Extensions.Logging.TraceSource/TraceSourceScope.cs [hunk 1]
```diff
@@ -14,11 +14,13 @@ internal class TraceSourceScope : IDisposable
         // To detect redundant calls
         private bool _isDisposed;
 
+#if NET451
```
Comment (NTaylorMullen) [inline src/Microsoft.Extensions.Logging.TraceSource/TraceSourceScope.cs]: Hmmm, should still have some sort of doc comments for non-net451. Ditto elsewhere

**Comment by NTaylorMullen** [comment]
:watch:

File: src/Microsoft.Extensions.Logging.TraceSource/TraceSourceScope.cs (hunk 1 above)
Comment (ajaybhargavb) [inline src/Microsoft.Extensions.Logging.TraceSource/TraceSourceScope.cs]: Added.

**Comment by ajaybhargavb** [comment]
:up: :date:

**Comment by NTaylorMullen** [comment]
:shipit:

**Pull request merged**